### PR TITLE
Really show My contributions-link

### PR DESCRIPTION
**Pull request by Elsensee**
This extends the previous PR #181 to actually make it work.

### CODE REVIEW

File: event/main_listener.php [hunk 1]
```diff
@@ -156,6 +156,18 @@ public function startup($event)
 
 	public function overwrite_template_vars($event)
 	{
+		if ($this->user->data['is_registered'] && !$this->user->data['is_bot'])
+		{
+			$u_my_contribs = $this->controller_helper->route('phpbb.titania.author', array(
```
Comment (michaelcullum) [inline event/main_listener.php]: If not in titania will this send to /customise/db/? I think not. @prototech

Comment (prototech) [inline event/main_listener.php]: It won't. 😞

Comment (Elsensee) [inline event/main_listener.php]: How often will you stab me if I use some kind of `str_replace($titania_config->phpbb_script_path, $titania_config->titania_script_path, ...);` here?

**Comment by michaelcullum** [comment]
Is this tested?

**Comment by Elsensee** [comment]
Yes.

**Pull request merged**